### DISCUSSION TITLE
"offline" use case - analysis of already recorded videos

### DISCUSSION THREAD

**Discussion by stepir** [blakeblackshear/frigate#4077] [Ask A Question]
Hi,

New user here, I'm just planning my first install. I have an i7 qnap with 32GB of ram, mostly running apps in containers (planning frigate in this way as well), and I just received my M2 Coral TPU.

I checked previous discussions and docs https://docs.frigate.video/contributing/#setup and it looks frigate can also be used to analyse mp4 files already been recorded. 

I have 10+ cameras already recording continuously on my NAS (+ doing some sort of alerting based on line crossing using embedded capabilities) and would like use the existing recordings to 'just' identify objects. 

Do you guys think Frigate can be used in such way? Anyone willing to share suggestions or thoughts on this case?

Thanks a lot

**Comment by NickM-27** [comment]
That's a convenient way to analyze existing footage if you have an issue and want to debug (like for a false positive). 

I don't think doing it just to run on old footage is advisable, it would be real time (not sped up detection) and I don't see a benefit to doing so.

**Comment by stepir** [comment]
hi @NickM-27 thanks a lot for the quick feedback. I think I can state better my use cases. I really don't want to analyse old videos, but would like Frigate to analyse the file recording being just closed on NAS by the cameras itself. Could it this be accomplished? I understand it would be post process and not real time.

I would use the analysis just in case I need to further investigate an event. I already have line-crossing working very well but sometimes I only wish all those videos would have some kind of search capabilities for other kind of events.

**Reply by NickM-27** [reply]
Oh I see now what you mean. No, frigate can certainly be made to record TO your NAS. 

Frigate does not really support nor do I think we are looking to support analyzing ongoing footage FROM a NAS.

**Reply by stepir** [reply]
got it thanks a lot for confirming!

**Comment by Xitee1** [comment]
@stepir have you found any solution?
I'm having the same "issue". I only want to have Frigate for the live camera overview and to analyse the existing videos on my NAS. My Reolink cameras already reliably record to my NAS if there is motion. It makes no sense recording everything twice. My cameras already detect the motion internally which is extremely reliable and it would not make sense for me to buy extra hardware (my Raspberry Pi with Home Assistant clearly can't handle permanent motion detection) to do something that is already working.

**Reply by Xitee1** [reply]
And I just like to have such an great overview of the recordings. Frigate gives me an clear overview what object, when it happened, which camera, and a preview image that actually shows the object.
I hate digging trough that big list of video files that only have the VLC icon with a timestamp in the filename..

**Reply by stepir** [reply]
nope unfortunately
